SQL-Query/Query-XII./SQLAnswer.md

```diff
@@ -0,0 +1,17 @@
+**12. Maximum units fulfilled by location:**
+```sql
+select 
+  oh.ORIGIN_FACILITY_ID, 
```

Comment:
This again should be the facility Id from the Order Item Ship Group.
And not Order Header's Origin Facility Id.